### PR TITLE
[#75] Adds another database state management service to apply migrations in the future.

### PR DESCRIPTION
**Pull request by lancelafontaine**
[#75] 

- [x] Adds another database state management service to apply migrations in the future.

### CODE REVIEW

File: src/nlp_service/services/mlService.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import requests
 
-from postgresql_db.models import Fact, FactEntity
+from db_state_service.models import Fact, FactEntity
```
Comment (arekmano) [inline src/nlp_service/services/mlService.py]: I personally prefer duplicating the models' code rather than breaking microservice architecture

Comment (lancelafontaine) [reply src/nlp_service/services/mlService.py]: This is something we should address as scope of work for issue #156 in iteration 5.

**Review by arekmano — APPROVED** [review]
LGTM

**Comment by lancelafontaine** [comment]
To be reopened again  in the future as part of issue #156

**Pull request closed without merging**